### ISSUE TITLE
Quantization of FastText

### ISSUE THREAD

**Issue by ivanpanshin** [facebookresearch/fastText#1119]
I have a question regarding quantization of FastText. As far as I understand, it only works for supervised models. But my end goal is to have good embeddings from FastText (not for some specific domain, but in general) + so that the model has gone through quantization. How can I do it? Train some classifier, then run quantization and hope that it's as good as pretrained model for general embeddings?

**Comment by minotru** [comment]
@ivanpanshin  Although it is a quite old question, someone might find this topic article :)
https://habr.com/ru/post/489474/

**Comment by ivanpanshin** [comment]
Yeah, read that (since I'm russian :) ), but that's for gensim, not this implementation.
